### PR TITLE
EPMFARMARS-12782

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: step_definitions/hooks.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"use strict";
```
Comment (SiarheiRylach) [inline step_definitions/hooks.js:1]: Please exclude report files from PR

File: features/delta_com_booking_sample.feature [hunk 2]
```diff
@@ -0,0 +1,41 @@
+@delta
+Feature: Delta.com Search results
+
+@smoke
+Scenario: Search results is empty 
+	Given I open "http://delta.com" url
+	Then I wait until element "toAirportName" of id type is visible
+		And I click on element "toAirportName" of id type
```
Comment (SiarheiRylach) [inline features/delta_com_booking_sample.feature:8]: Indentation?

File: features/delta_com_booking_sample.feature [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	Then I enter "Berlin" in element "search_input" of id type and hit Enter key
+	Then I wait until element "li.airport-list:nth-child(1)" of css type is present
```
Comment (SiarheiRylach) [inline features/delta_com_booking_sample.feature:10]: not good idea, but accepted for now

File: features/delta_com_booking_sample.feature [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+		And I click on element "li.airport-list:nth-child(1)" of css type
+	Then I wait until element "selectTripType-val" of id type is present
+		And I click on element "selectTripType-val" of id type
```
Comment (SiarheiRylach) [inline features/delta_com_booking_sample.feature:13]: Use of When and Then

File: step_definitions/browser_steps.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+const { When, Then } = require('cucumber');
+const { browser } = require('protractor');
+const { expect } = require('chai');
+const elementHandler = require('../utils/page_elements_handler');
+
+When(/^I open "([^"]*)" url$/, (url) => {
+    console.log(`I open ${url} url`);
```
Comment (SiarheiRylach) [inline step_definitions/browser_steps.js:7]: logger?

File: step_definitions/hooks.js (hunk 1 above)
Comment (overdd) [reply step_definitions/hooks.js:1]: Fixed

File: features/delta_com_booking_sample.feature (hunk 2 above)
Comment (overdd) [reply features/delta_com_booking_sample.feature:8]: Fixed with Gherkin Indent plug-in

File: features/delta_com_booking_sample.feature (hunk 3 above)
Comment (overdd) [reply features/delta_com_booking_sample.feature:10]: Agree

File: features/delta_com_booking_sample.feature (hunk 4 above)
Comment (overdd) [reply features/delta_com_booking_sample.feature:13]: Fixed

File: step_definitions/browser_steps.js (hunk 5 above)
Comment (overdd) [reply step_definitions/browser_steps.js:7]: Removed

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**